modules/javafx.graphics/src/main/java/com/sun/javafx/font/CompositeGlyphMapper.java

```diff
@@ -100,7 +100,6 @@ private final int convertToGlyph(int unicode) {
                 return glyphCode;
             }
         }
-        glyphMap.put(unicode, missingGlyph);
```

Comment:
Does removing this entry from the map have any side effects?

Reply:
AFAIK, it shouldn't have side effect. @prrace , what is your opinion?